tests/fixtures/custom_pred_consistency/custom.py

```diff
@@ -0,0 +1,127 @@
+import pandas as pd
+import numpy as np
```

Comment:
could you please tidy this? thanks.

Reply:
I added some docstrings to explain what's going on - this turns out to be a pretty good example of how to separately build a preprocessing pipeline and a model and tell drum how to load/run them where needed rather then bundling them into one pkl object